src/error.rs

```diff
@@ -16,8 +16,6 @@ pub enum Error {
         src_line_number: u16,
     },
     InvalidTabCall,
-    UndefinedNumericVariable(NumericVariable),
-    UndefinedStringVariable(StringVariable),
```

Comment:
So these errors go?

Reply:
Ah, yes. They go, because the test actually tests that undefined variables by default are 0 resp. "". This is apparently due to ANSI spec. ECMA spec is silent about it.